README.md

```diff
@@ -138,12 +138,23 @@ Cut out the squares and put the segments of code in order in which they are ran,
 - When button, 'Make Reservation', is clicked
 
 ### Initial Page Load
-When the page initially loads, there is a lot of code in various parts of our application that are depended on to make sure it all comes together.
+**Activity:**
+When the page initially loads, there is a lot of code in various parts of our application that are depended on to make sure it all comes together. To help understand how this all fits together, let's leave ourselves a trail of breadcrumbs to follow.
+
+To do so, we'll add a log line to the top of every function in our TREK app. Each line should be of form
+
+```javascript
+console.log(">>> BREADCRUMBS: #");
+```
+
+Where `#` is the order in which the log line appears when the app is started. If the function shouldn't be called when the page is first loaded, replace `#` with `does not print`.
+
```

Comment:
I would suggest an example:

So for the first function called in the application you should place:  `console.log(">>> BREADCRUMBS: 1");`